libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsTextPlayer.kt

```diff
@@ -102,9 +102,13 @@ internal class VoiceInstructionsTextPlayer(
         volumeLevel = DEFAULT_VOLUME_LEVEL
     }
 
-    private fun initializeWithLanguage(language: Locale) {
-        isLanguageSupported =
+    @VisibleForTesting
+    internal fun initializeWithLanguage(language: Locale) {
+        isLanguageSupported = if (playerAttributes.options.checkIsLanguageAvailable) {
             textToSpeech.isLanguageAvailable(language) == TextToSpeech.LANG_AVAILABLE
+        } else {
+            true
+        }
```

Comment:
This is a bit counterintuitive as we're saying that if `checkIsLanguageAvailable` is `false`, `isLanguageSupported` is `true` but I understand why we're doing it, so that we don't return here https://github.com/mapbox/mapbox-navigation-android/blob/fdfd1ab7512f9e38577c55310319553249b0a945/libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/VoiceInstructionsTextPlayer.kt#L112-L115 right?

Reply:
@Guardiola31337 

The checkIsLanguageAvailable flag says whether to check for language availability or not. There are four possible cases
| Language Supported by TTS | checkIsLanguageAvailable | Result |
| ------------- | ------------- | ------------- |
| true  | true  | Flag `isLanguageSupported` will be true. TTS will play ssml |
| false  | true  | Flag `isLanguageSupported` will be false. TTS will skip ssml with error LANGUAGE_NOT_SUPPORTED or announcement from state is blank  |
| true  | false  | Flag `isLanguageSupported` will be true. TTS will play ssml   |
| false  | flase  | Flag `isLanguageSupported` will be true. Unexpected behaviour  |

Not sure what I should fix on this side

Reply:
Nothing to fix here. Thanks for the clarification 👍